File: CODES/4) Php/10) MASSIF/21.php
<?php
$n = 10;
$K = 5;
$L = 7;
for ($i = 0; $i < $n; $i++) {
    $a[$i] = rand() % 100;
    echo "A[" . $i  . "] = " . $a[$i] . "<br>";
}
echo "<br>";
for ($i = $K; $i <= $L; $i++) {
    echo "A[" . $i  . "] = " . $a[$i] . "<br>";
    $S += $a[$i];
    $c++;
}
echo " O'rta arifmetigi = " . $S / $c;

File: CODES/1) C++/11) MASSIF 2 O'LCHAMLI/21.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){
	int m,n;
	float b, c=0;
	cin>>m>>n;
	int a[m][n];
	for(int i=0;i<m;i++){
		for(int j=0;j<n;j++){
			cout<<"A["<<i<<"]["<<j<<"]=";
			cin>>a[i][j];
		}
	}
  for(int i=0;i<m;i++){
    for(int j=0;j<n;j++){
      cout<<a[i][j]<<" ";
    }
    cout<<endl;
  }
  for(int i=0;i<m;i++){
    for(int j=0;j<n;j++){
    	if(j%2==1){
    		b+=a[i][j];
    		c++;
		}
    }
     cout<<endl<<i+1<<"-satr o'rta arifmetigi = "<<b/c;
     b=0;
     c=0;
  }
return 0;
}